oauth-common/src/main/java/io/strimzi/kafka/oauth/services/Services.java

```diff
@@ -39,6 +39,13 @@ public static synchronized void configure(Map<String, ?> configs) {
         }
     }
 
+    /**
+     * Close any configured Services so they can be reinitialised again
+     */
+    public static synchronized void close() {
+        services = null;
```

Comment:
This is not really _closing_ services

Reply:
Maybe one day it will be. There are at the moment no threads running in there so I simply release the memory. You think this should be called differently or the docs should be different or you think the implementation should be different?